audb/core/publish.py

```diff
@@ -262,18 +356,28 @@ def publish(
     r"""Publish database.
 
     A database can have dependencies
-    to files of an older version of itself.
-    E.g. you might add a few new files to an existing database
-    and publish as a new version.
-    :func:`audb.publish` will upload then only the new files
-    and store dependencies on the already published files.
-
-    To allow for dependencies
-    you first have to load the version of the database
+    to media files and tables of an older version.
+    E.g. you might alter an existing table
+    by adding new media files to it
+    and publish it as a new version.
+    :func:`audb.publish` will then upload
+    only the altered table
+    and the new media files,
+    which will be added as new dependencies
+    to the already published files.
+
+    To update a database,
+    you first have to load the version
```

Comment:
That's not completely true as you can also start from scratch again for a new database version, but I think it is ok if we go with your statement as it is correct in 99.9% of all cases.

Reply:
This case is covered in the next section:

![image](https://user-images.githubusercontent.com/10383417/178918814-8df2c50a-bf09-44ef-8e03-d9970793aa6e.png)

Reply:
Ah, nice.